network_wrangler/models/gtfs/tables.py

```diff
@@ -318,7 +318,7 @@ class Config:
 class WranglerStopTimesTable(StopTimesTable):
     """Wrangler flavor of GTFS StopTimesTable."""
 
-    model_node_id: Series[int] = pa.Field(coerce=True)
```

Comment:
Stop_times table doesn’t have model_node_id column. Can remove it. Otherwise it will add a new column of NaN to the table. And it will cause error in shape_gtfs_to_cube() in metcouncil_wrangler

Reply:
Yeah - we need to resolve this re this dicussion https://github.com/wsp-sag/network_wrangler/issues/354